extensions/podman/src/extension.ts

```diff
@@ -726,9 +733,24 @@ export async function activate(extensionContext: extensionApi.ExtensionContext):
   await podmanConfiguration.init();
 }
 
+export async function stopMachine(machineName: string, machineStarted: boolean) {
+  // do we have an autostarted machine ?
+  if (!machineStarted) {
```

Comment:
The machineStarted always receive the value of the autoStart. So if you stopped the machine manually before closing podman desktop and you have the autostart set to true, when it comes here it tries to stop a machine already stopped.

Reply:
Attempting to stop a machine that is already in a stopped state does not result in an error, so there is no issue with doing so.  If you think it would be beneficial to add a check to improve the efficiency of the function, I can make the change.